lib/lndclient/LndClient.ts

```diff
@@ -210,6 +210,22 @@ class LndClient extends SwapClient {
     this.emit('locked');
   }
 
+  /** Lnd specific procedure to mark the client as unlocked. */
+  private setUnlocked = () => {
+    // we should close and unreference the wallet unlocker service when we set the status to Unlocked
```

Comment:
I'm glad we won't have to deal with this soon when Lnd moves to single service for all calls.

Reply:
Oh yeah that would be good, where have you been following that progress? We might want to do the same as well, I had based the separate unlock service off lnd but then the more I've had to deal with it (and after discussing issues in the grpc repos) I'm realizing having grpc service per port is the way to go.

Reply:
https://github.com/lightningnetwork/lnd/issues/3982